posthog/temporal/batch_exports/batch_exports.py

```diff
@@ -258,7 +278,17 @@ async def iter_records_from_model_view(
         else:
             parameters["include_events"] = []
 
-        if str(team_id) in settings.UNCONSTRAINED_TIMESTAMP_TEAM_IDS:
+        start_at = dt.datetime.fromisoformat(interval_start) if interval_start is not None else None
+        end_at = dt.datetime.fromisoformat(interval_end)
+
+        if start_at:
+            is_5_min_batch_export = (end_at - start_at) == dt.timedelta(seconds=300)
+        else:
+            is_5_min_batch_export = False
+
+        if is_5_min_batch_export and not is_backfill:
+            query_template = SELECT_FROM_EVENTS_VIEW_RECENT
```

Comment:
We can't do backfills from here as table will not be backfilled (initially at least). But existing table should be without lag by the time users decide to backfill.